app/models/hardware.rb

```diff
@@ -51,13 +51,12 @@ def mac_addresses
     @mac_addresses ||= nics.collect(&:address).compact.uniq
   end
 
-  def ram_size_in_bytes
-    memory_mb.to_i * 1.megabyte
+  def memory_mb
```

Comment:
may want to add a `memory_mb=` and deprecate it since that is possibly called by others